app/controllers/user_application_controller.rb

```diff
@@ -0,0 +1,13 @@
+require './app/services/builders/user_navigation_builder.rb'
```

Comment:
Rails should autoload this class, maybe there should be a single configuration in application.rb for that